JustSaying.AwsTools.IntegrationTests/WhenSettingUpMultipleHandlers.cs

```diff
@@ -81,21 +81,15 @@ protected override void When()
         }
 
         [Test]
-        public void CreateTopicCalledOnce()
-        {
-            Assert.That(proxyAwsClientFactory.Counters["CreateTopic"][topicName].Count, Is.EqualTo(1));
-        }
-
-        [Test]
-        public void FindTopicCalledOnce()
+        public void CreateTopicCalled()
         {
-            Assert.That(proxyAwsClientFactory.Counters["FindTopic"][topicName].Count, Is.EqualTo(1));
+            Assert.That(proxyAwsClientFactory.Counters["CreateTopic"][topicName].Count, Is.GreaterThanOrEqualTo(1));
         }
 
         [Test]
-        public void ListQueuesCalledOnce()
```

Comment:
testing the implementation detail and API calls like this lead to brittle test but have updated existing assertions as we don't have the integration tests running under CI yet